### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by gssbzn**
## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,71 +84,29 @@ or [Cloud Manager](https://docs.cloudmanager.mongodb.com/tutorial/manage-program
...
-```bash
-source <(mongocli completion zsh)
-``` 
-
-To load completions for each session, execute once:
-
-```bash
-mongocli completion zsh > "${fpath[1]}/_mongocli"
-```
+If you install via [homebrew](#hombrew-on-macos) there's nothing else to do.
```
Comment (viviandiancheng) [inline README.md:99]: Do you mean installing via homebrew, users get completion by default and don't need to take extra steps? If so, maybe change to:
If you install via [homebrew](#hombrew-on-macos) no additional actions are needed.

**Review by viviandiancheng — COMMENTED** [review]
Minor copy suggestions.

File: README.md [hunk 2]
```diff
@@ -6,7 +6,16 @@
 ![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
 ![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
 
-`mongocli` is a tool for managing your MongoDB cloud services
+The MongoDB CLI is a modern command line interface that enables you to manage your MongoDB services from the terminal.
+
+![quickstart](https://user-images.githubusercontent.com/461027/126986233-0dd5c82a-2c75-4887-ab66-eb018c59e093.gif)
```
[the remark this answers is not in the document]
Comment (gssbzn) [reply README.md:11]: yes, I meant to fix this and then forgot

**Review by viviandiancheng — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -6,7 +6,16 @@
 ![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
 ![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
 
-`mongocli` is a tool for managing your MongoDB cloud services
+The MongoDB CLI is a modern command line interface that enables you to manage your MongoDB services from the terminal.
```
Comment (threebee) [inline README.md:9]: [q] Should it be `MongoCLI` instead of `MongoDB CLI`? I got bit confused by reading `MongoDB CLI`.

File: README.md [hunk 4]
```diff
@@ -65,7 +75,7 @@ or check our [online documentation](https://docs.mongodb.com/mongocli/master/) f
 
 ### Getting API Keys
 
-To use mongocli you'll need to get API keys, to do this please follow the documentation 
+To use `mongocli` you'll need to get API keys, to do this follow the documentation
```
Comment (threebee) [inline README.md:78]: [suggestion] `to do this` -> `to get one`

**Review by threebee — DISMISSED** [review]
LGTM, I have added a question and a suggestion. Nothing blocking.

File: README.md (hunk 3 above)
Comment (gssbzn) [reply README.md:9]: Official name MongoDB CLI, `mongocli` for short

**Review by threebee — APPROVED** [review]
LGTM, thank you for the changes! :shipit:

**Pull request merged**